ng/src/web/entities/table.js

```diff
@@ -106,6 +106,13 @@ class EntitiesTable extends React.Component {
 
     const filterstring = is_defined(filter) ? filter.toFilterString() : '';
 
+    let sortDir;
+    let filterSortBy;
+    if (is_defined(filter)) {
```

Comment:
Either put also line 107 into the if clause or use the ternary operator also for sortDir and filterSortBy/sortDirection